components/NavigationBar/socialIcons.js

```diff
@@ -0,0 +1,24 @@
+export const socialIcons = [
```

Comment:
Please move this to a constants (e.g. `constats/icons.js`)

Reply:
done